src/GraphQL/Execution/ExecutionOptions.cs

```diff
@@ -12,22 +12,39 @@
 
 namespace GraphQL
 {
+    /// <summary>Configuration options to be passed to <see cref="IDocumentExecuter"/> to execute a query</summary>
     public class ExecutionOptions : IProvideUserContext
     {
+        /// <summary>Schema of graph to use; required<br/><br/>Schema will be initialized if has not yet been initialized.</summary>
         public ISchema Schema { get; set; }
+
+        /// <summary>Object to pass to the <see cref="IResolveFieldContext.Source"/> property of first-level resolvers</summary>
         public object Root { get; set; }
+
+        /// <summary>GraphQL query to parse and execute; required</summary>
         public string Query { get; set; }
+
+        /// <summary>GraphQL query operation name; defaults to first operation defined in query</summary>
         public string OperationName { get; set; }
+
+        /// <summary>Parsed GraphQL request; will parse <see cref="Query"/> if not set</summary>
         public Document Document { get; set; }
+
+        /// <summary>Input variables to GraphQL request</summary>
         public Inputs Inputs { get; set; }
-        
+
+        /// <summary><see cref="System.Threading.CancellationToken">CancellationToken</see> of request; optional</summary>
```

Comment:
```suggestion
        /// <summary><see cref="System.Threading.CancellationToken">CancellationToken</see> to cancel the request at any stage of its execution; defaults to <see cref="System.Threading.CancellationToken.None"/></summary>
```

Reply:
If you think this is a good change, I don't mind changing it.  CancellationToken.None is the same as leaving it undefined, as CancellationToken is a struct.  So I think of it as an optional parameter.

I like this part for sure: `to cancel the request at any stage of its execution`

Reply:
Seems fine anyway however...

Reply:
I know that it is struct but despite this we could set this property to some value other than None. For example, we could by default set cancellation after 60 seconds. Therefore, it’s better to clearly say that it is equal to None.